samples/SignalRSamples/Startup.cs

```diff
@@ -62,6 +65,41 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 routes.MapConnectionHandler<MessagesConnectionHandler>("/chat");
             });
+
+            app.Use(next => (context) =>
+            {
+                if (context.Request.Path.StartsWithSegments("/deployment"))
+                {
+                    var attributes = Assembly.GetAssembly(typeof(Startup)).GetCustomAttributes<AssemblyMetadataAttribute>();
+
+                    context.Response.ContentType = "application/json";
+                    using (var textWriter = new StreamWriter(context.Response.Body))
+                    using (var writer = new Newtonsoft.Json.JsonTextWriter(textWriter))
```

Comment:
Creating a JObject and then writing it with `o.WriteTo(output)` is simpler. We only use the low-level `JsonTextWriter` in SignalR for extra perf.

Will this JSON ever change? If it will be something that will be touched again in the future it might be worth making it easier to modify.

Reply:
Yeah, the JObject syntax is probably much cleaner :)